src/domain/group/component/InputGroup.js

```diff
@@ -64,19 +65,26 @@ class InputGroup extends React.Component {
               return (
                 <form className="form-inline" key={key}>
                   {matches ? [
-                    <label style={{ marginBottom: '0', marginRight: '.5rem' }} key="label">Group ID</label>,
+                    <label style={{ marginBottom: '0', marginRight: '.5rem' }} key={uuid.v4()}>Group ID</label>,
```

Comment:
@hellowin https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318